src/state/user/reducer.ts

```diff
@@ -74,6 +78,8 @@ export const initialState: UserState = {
   isDark: false,
   userFarmStakedOnly: FarmStakedOnly.ON_FINISHED,
   gasPrice: GAS_PRICE_GWEI.default,
+  watchlistTokens: [],
```

Comment:
Update to existing code:
Storing watchlistTokens and watchlistPools in `user` reducer